src/services/ghost/new-auto-complete/NewAutocompleteModel.ts

```diff
@@ -49,6 +49,7 @@ export class NewAutocompleteModel {
 				if (!(await checkKilocodeBalance(profile.kilocodeToken, profile.kilocodeOrganizationId))) continue
 			}
 
+			this.profile = profile
```

Comment:
sure? the code around here seems to sometimes use this and sometimes not; it also sometimes uses this.apihandler and sometimes this.profile.apiHandler